### PR TITLE
docs: add cursor ide

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Fixes https://github.com/daytonaio/docs/issues/133

### CODE REVIEW

**Review by Tpuljak — APPROVED** [review]

**Comment by Tpuljak** [comment]
@stefanicjuraj lets get the conflicts resolved so we can merge

File: src/content/docs/about/getting-started.mdx [hunk 1]
```diff
@@ -193,7 +193,11 @@ Use the `daytona target --help` command to view available configuration options
 
 ## Choosing the Default IDE
 
+<<<<<<< #133-add-cursor-ide
```
Comment (Tpuljak) [inline src/content/docs/about/getting-started.mdx:196]: Not sure how this even got through 😅

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Review by Tpuljak — APPROVED** [review]

**Comment by stefanicjuraj** [comment]
Hi, seems to be working fine locally for me - in the previous pull requests we changed the base path of the oss documentation to `/docs`, so maybe the recent changes weren't propagated for you? - referring to this: https://github.com/daytonaio/docs/pull/140

image

**Review by ivan-burazin — APPROVED** [review]

**Comment by Tpuljak** [comment]
@stefanicjuraj rebase with main and we're good to go

**Pull request merged**